srcpkgs/blockify/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'blockify'
+pkgname=blockify
+version=3.6.3
+revision=1
+#archs="i686 x86_64"
```

Comment:
Get rid of commented out lines

Reply:
got it

Reply:
> (╯°□°）╯︵ ┻━┻
> 
> ```
> spotify-client_1.1.42.622.gbd112320-37_amd64.deb: [120MB 0%] 100MB/s ETA: 00m00s
> spotify-client_1.1.42.622.gbd112320-37_amd64.deb: [120MB 0%] 141KB/s ETA: 24m27s
> spotify-client_1.1.42.622.gbd112320-37_amd64.deb: [120MB 0%] 188KB/s ETA: 13m51s
> spotify-client_1.1.42.622.gbd112320-37_amd64.deb: [120MB 0%] 252KB/s ETA: 09m31s
> spotify-client_1.1.42.622.gbd112320-37_amd64.deb: [120MB 0%] 339KB/s ETA: 06m45s
> spotify-client_1.1.42.622.gbd112320-37_amd64.deb: [120MB 1%] 467KB/s ETA: 04m46s
> http://repository.spotify.com/pool/non-free/s/spotify-client/spotify-client_1.1.42.622.gbd112320-37_amd64.deb: (null)
> => ERROR: spotify-1.1.42_2: failed to fetch spotify-client_1.1.42.622.gbd112320-37_amd64.deb.
> ```

Oof, I'd just try rebasing and pushing tomorrow.

Reply:
You can resolve the conversation so that it's closed now ~